opensearchutil/bulk_indexer.go

```diff
@@ -114,7 +113,10 @@ type BulkIndexerItem struct {
 	Index           string
 	Action          string
 	DocumentID      string
-	Body            io.Reader
+	Routing         string
+	Version         int64
```

Comment:
Why not make those optional fields as pointer?
```suggestion
	Version         *int64
```